tests/useCases/pedido/index.test.ts

```diff
@@ -163,11 +171,11 @@ describe("Given PedidoUseCases", () => {
         });
     });
 
-    describe("When create is called", () => {
-        it("should call create on the gateway and return the created pedido", async () => {
-            const create = jest.spyOn(gatewayStub, "create");
+    describe("When checkout is called", () => {
+        it("should call create on the gateway and return the created pedido id", async () => {
```

Comment:
Micro detalhe, só alterar o texto desse teste
```suggestion
        it("should call checkout on the gateway and return the created pedido id", async () => {
```